src/components/menu/MenuHeader.tsx

```diff
@@ -7,13 +7,15 @@ import MenuContext from './MenuContext';
 import './MenuHeader.scss';
 
 export interface MenuHeaderProps {
-    /** children - menu section header content */
-    children?: Array<React.ReactChild> | React.ReactChild;
```

Comment:
Might be a good idea to keep `children` as well in case we need to customize the menu header in the future.